danielMunteanu/src/main/java/danielCollectionTask/FilterByCity.java

```diff
@@ -0,0 +1,21 @@
+package danielCollectionTask;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class FilterByCity {
+    public static List<User> filterUsersByCity(List<User> users, String city) {
```

Comment:
How about AdminUser, still we need one more method or we can  use this one? What changes should be done? Let me know if more details are need.

Reply:
reworked as abstract